tangelo/toolboxes/molecular_computation/tests/test_psi4.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2023 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+
+from tangelo import SecondQuantizedMolecule
+from tangelo.helpers.utils import installed_chem_backends
+from tangelo.problem_decomposition.oniom.oniom_problem_decomposition import ONIOMProblemDecomposition
+from tangelo.problem_decomposition.oniom._helpers.helper_classes import Fragment
+from tangelo.toolboxes.molecular_computation.integral_solver_psi4 import IntegralSolverPsi4
+from tangelo.algorithms.variational import SA_OO_Solver, BuiltInAnsatze, ADAPTSolver, iQCC_solver
+from tangelo.molecule_library import xyz_H4, mol_H4_minao, xyz_H2, mol_H4_sto3g_uhf_a1_frozen
+
+
+class Testpsi4(unittest.TestCase):
+
+    @unittest.skipIf("psi4" not in installed_chem_backends, "Test Skipped: Backend not available \n")
```

Comment:
Just use the decorator on the whole class instead of repeating it per test
example: https://github.com/goodchemistryco/Tangelo/blob/main/tangelo/linq/tests/test_ibm_connection.py